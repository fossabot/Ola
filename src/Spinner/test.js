import React from 'react'
import { Spinner } from './'
import renderer from 'react-test-renderer'

it('Default Spinner', () => {
  const tree = renderer
    .create(<Spinner  />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Small Spinner', () => {
  const tree = renderer
    .create(<Spinner size="small" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Medium Spinner', () => {
  const tree = renderer
    .create(<Spinner size="medium" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Big Spinner', () => {
  const tree = renderer
    .create(<Spinner size="big" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Spinner with extraClasss', () => {
  const tree = renderer
    .create(<Spinner extraClasss="extraclass" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
