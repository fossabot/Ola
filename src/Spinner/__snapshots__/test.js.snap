// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Big Spinner 1`] = `
<svg
  className="ola_spinner is-big"
  height={140}
  viewBox="0 0 140 140"
  width={140}
>
  <circle
    cx={70}
    cy={70}
    r={62}
  />
</svg>
`;

exports[`Default Spinner 1`] = `
<svg
  className="ola_spinner"
  height={28}
  viewBox="0 0 28 28"
  width={28}
>
  <circle
    cx={14}
    cy={14}
    r={12}
  />
</svg>
`;

exports[`Medium Spinner 1`] = `
<svg
  className="ola_spinner is-medium"
  height={76}
  viewBox="0 0 76 76"
  width={76}
>
  <circle
    cx={38}
    cy={38}
    r={34}
  />
</svg>
`;

exports[`Small Spinner 1`] = `
<svg
  className="ola_spinner"
  height={28}
  viewBox="0 0 28 28"
  width={28}
>
  <circle
    cx={14}
    cy={14}
    r={12}
  />
</svg>
`;

exports[`Spinner with extraClasss 1`] = `
<svg
  className="ola_spinner"
  extraClasss="extraclass"
  height={28}
  viewBox="0 0 28 28"
  width={28}
>
  <circle
    cx={14}
    cy={14}
    r={12}
  />
</svg>
`;
